codec/codec-sofa-hessian/src/test/java/com/alipay/sofa/rpc/codec/sofahessian/SofaHessianSerializerTest.java

```diff
@@ -81,6 +81,14 @@ public void encode() {
         Assert.assertTrue(error);
     }
 
+    @Test
```

Comment:
This test is not associate with other codes.

Reply:
fixed